readthedocs/projects/models.py

```diff
@@ -1906,184 +1906,204 @@ def add_features(sender, **kwargs):
     HOSTING_INTEGRATIONS = "hosting_integrations"
 
     FEATURES = (
-        (ALLOW_DEPRECATED_WEBHOOKS, _('Allow deprecated webhook views')),
+        (ALLOW_DEPRECATED_WEBHOOKS, _("Webhook: Allow deprecated webhook views")),
         (
             DONT_OVERWRITE_SPHINX_CONTEXT,
             _(
-                'Do not overwrite context vars in conf.py with Read the Docs context',
+                "Sphinx: Do not overwrite context vars in conf.py with Read the Docs context",
             ),
         ),
         (
             SKIP_SPHINX_HTML_THEME_PATH,
             _(
-                "Do not define html_theme_path on Sphinx < 6.0",
+                "Sphinx: Do not define html_theme_path on Sphinx < 6.0",
             ),
         ),
         (
             MKDOCS_THEME_RTD,
-            _('Use Read the Docs theme for MkDocs as default theme'),
+            _("MkDocs: Use Read the Docs theme for MkDocs as default theme"),
         ),
         (
             DONT_SHALLOW_CLONE,
-            _('Do not shallow clone when cloning git repos'),
+            _("Git: Do not shallow clone when cloning git repos"),
         ),
         (
             USE_TESTING_BUILD_IMAGE,
-            _('Use Docker image labelled as `testing` to build the docs'),
+            _("Build: Use Docker image labelled as `testing` to build the docs"),
         ),
         (
             API_LARGE_DATA,
-            _('Try alternative method of posting large data'),
+            _("API: Try alternative method of posting large data"),
```

Comment:
Wasn't really sure of this one

Reply:
This is used from the builds, so it can be `Build`, but I'm fine with either one.

Reply:
Great, I've called it Build instead :+1: